### ISSUE TITLE
Required Field (Option / Radio) not actually required...

### ISSUE THREAD

**Issue by zerafox** [gameserverapp/Platform#763]
__Please make sure to leave out any sensitive information__!

**Describe the bug**
Required Field for atleast Option/Radio in form is not actually requiring users to complete.

**To Reproduce**
1. Create a Form
2. Set a Option / Radio and require it 
3. Do not fill out this option in form and press Send/Complete

**Expected behavior**
I expect the form to not allow it to send/complete as the required field was not actually filled out

**Screenshots**
https://gyazo.com/33be017f7675b824c4094a365d71b2fc

**Additional context**
I did not test any other forum functions to see if these are also affected.

**Comment by mvaessen** [comment]
Thanks for reporting @zerafox, this should now be fixed. Please give it another try and let me know if it works now or not.

**Comment by mvaessen** [comment]
hi, can you provide a screenshot of your form settings so i can reproduce?

**Comment by zerafox** [comment]
https://gyazo.com/f65b7007bd65937f05a2b83dab873462

I then attempted to switch them both to required to see if that would make a difference and entered both text and selected an option.
Regardless of what I was doing, the forum could not be submitted.

I did not try without any of the fields being required though, I didn't think that was needed. XD

**Comment by mvaessen** [comment]
I created a form exactly like the one from your screenshot and was not able to reproduce at first. But after some trial and error I noticed that I was able to reproduce the issue when submitting the form shortly after it was updated.

The community website will cache the page & form and after updating the form, the ID's for the fields will change.
The problem is that the page cache is only removed when the page is saved, but not when saving a form. I'm not sure if I'm able to add that, due to how stuff is set up. 

So in short: when updating a form, GSA might not show the updated form fields immediately on the community website. It may take a couple minutes for it to update. A work around is fairly easy: saving the page on which the form is embedded will purge the cache.

**Comment by mvaessen** [comment]
i've created a new issue for this specific issue (#766) and will continue the fix for the cache problem there. Thanks for the detailed info and retesting!
